dnsleaktest_FreeBSDjail.sh

```diff
@@ -1,4 +1,8 @@
-#!/bin/bash
```

Comment:
Please use the following code

`#!/usr/bin/env bash
`

please test the suggested code in your environment.

Reply:
Yes, that's better. Confirmed working a freebsd jail with pkg bash installed.